fix(parser): Skip `id` attribute from normalization

miroljub1995 opened the pull request:
<!--
        Hi there!
        Thanks for taking the time and putting the effort into making fabric better! 💖
        Take a look at /CONTRIBUTING.md for crucial instructions regarding local setup, testing etc.
        https://github.com/fabricjs/fabric.js/blob/master/CONTRIBUTING.md

        Adding tests that verify your fix and safeguard it from unwanted loss and changes is a MUST.

        Pull Requests are not always simple. Don't hesitate to ask for help (beware of [gotchas](http://fabricjs.com/fabric-gotchas) 😓).
        We appreciate your effort and would like the process to be productive and enjoyable.
        A strong community means a strong and better product for everyone.

        Each PR can introduce a single feature or change or fix a single bug
        Large refactors PR have been discussed before and probably splitted in steps
-->

## Description

<!-- Summarize the reasons of your changes and the meaning of your code. Why the code you are changing is wrong? why your is better? -->
<!-- If you are fixing a regression, please link the commit that introduced the bug -->
<!-- If you are proposing a feature, please link the discussion/issue where that was presented and discussed -->
<!-- Is this pullrequest a follow up from an open issue? if so please link the issue like the example below in addition to the summary -->
<!-- Related to #1234 -->

If we load SVG that has some object with `id` attribute starting with a `number`, we get a wrong id parsed.

zhe-he commented:
This kind of attribute has issues as well, right?
`<svg data-custom-attribute="123abc"></svg>`

asturur commented on src/parser/normalizeValue.ts:
```diff
@@ -56,7 +56,12 @@ export function normalizeValue(
+    attr === 'xlink:href' ||
+    attr === 'font' ||
+    attr === 'id'
+  ) {
```
this function is a bit weird.
We do carry a parse and and outputValue around and in some cases we just do an early return.
Hopefully there is a better way to fix it. All the attributes that start with a number that are not specially taken care of here are going to be wrong correct?
So any custom attribute will need to be listed here and that is not really feasible

miroljub1995 replied:
is it better to explicitly specify which one needs to be normalized?

asturur replied:
somehow this function become really convoluted yesterday i tried to look at it to expedite this fix. Eventually we can just fix as you did and open a ticket for a refactor, i didn't decide it yet sorry.

asturur commented:
@miroljub1995 add this line to the changelog:
fix(SVGImport): Allow parsing of 'id' attribute that starts with a number [#10079](https://github.com/fabricjs/fabric.js/pull/10079)

and we can merge.

Please check that it gets formatted as the other lines or we get a lint error.
I can't edit myself as i don't have write access to your branch

miroljub1995 commented:
@asturur 
- I added line to CHANGELOG
- Regarding to lint, I checked with `npm run prettier:check && npm run lint`, but there are already some errors so I just checked if no new errors are on files I changed.

asturur commented:
There are maybe warnings, the checkmark is green.
Or maybe somehow you are picking up a different configuration
